packages/shared/src/styles/constants.scss

```diff
@@ -196,3 +196,8 @@ $gradient-webmoney: linear-gradient(90deg, #1a77ac 0%, #0068a3 100%);
 /* App Cards gradient background */
 $gradient-virtual: linear-gradient(274.25deg, #333333 9.01%, #5c5b5b 103.31%);
 $gradient-virtual-swap-free: linear-gradient(58.51deg, #061828 28.06%, #1a3c60 93.51%);
+
+/* Wallets */
+$wallets-banner-box-shadow: 0px 12px 16px -4px rgba(14, 14, 14, 0.08), 0px 4px 6px -2px rgba(14, 14, 14, 0.03);
```

Comment:
do not need to define this shadow since it's a unique case

Reply:
Two things here:
- First when I comment on using the variables, I mean the existing ones, we have variables for `box-shodaw` or `border-radius`. We need to use them.
- Also, I think the `shared` is not a good place to store app store variables.

P.S: If the existing `box-shadow` variable is different from the design, you can use the value itself, as @mahdiyeh-deriv suggested. However, design should have consistency for these kinds of values.